assets/app/styles/_core.less

```diff
@@ -104,6 +104,16 @@ html, body {
       margin-bottom: 0;
     }
   }
+  .label-filter {
```

Comment:
would it make sense to put the `.label-filter` and `.active-filters` changes in a PR to https://github.com/kubernetes-ui/label-selector ?